.circleci/config.yml

```diff
@@ -53,7 +53,12 @@ jobs:
       - run:
           name: docker image build
           command: |
-            TAG=`cat ./.tag`
+            if [ ! -z "$CIRCLE_TAG" ]; then
```

Comment:
should the logic is something like 

```
if ./.tag file found 
    use the tag file version
else 
    ?? not sure the handling 
```

Reply:
CIRCLE_TAG has priority.
On `./.tag` not found,`TAG` is empty.
the below if else will do the checking

Reply:
ok